lib/Sema/CSDiag.cpp

```diff
@@ -3345,6 +3352,22 @@ class ArgumentMatcher : public MatchCallArgumentListener {
     Diagnosed = true;
   }
 
+  bool isPropertyWrapperImplicitInit() {
+    if (auto TE = dyn_cast<TypeExpr>(FnExpr)) {
+      if (TE->getInstanceType()
+              ->getAnyNominal()
```

Comment:
Just to be safe let's check whether `TypeExpr` has a type associated with it first and if it's a nominal type.

Reply:
@xedin Done!